src/App.jsx

```diff
@@ -14,70 +15,70 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+const goodsObj = goods
+  .map((item, index) => ({ nameGoods: item, id: index + 1 }));
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export const App = () => {
+  const [checked, setChecked] = useState(9);
+  const [currentName, setCurrentName] = useState('Jam');
```

Comment:
```suggestion
  const [selectedGood, setSelectedGood] = useState('Jam');
```
There should be only one value in state
![image](https://github.com/mate-academy/react_goods-selector-js/assets/96409563/8a17093d-08a9-448a-8e5b-e4e0b65a5986)